modules/n1ql/pages/n1ql-language-reference/typefun.adoc

```diff
@@ -59,6 +55,14 @@ SELECT ISARRAY(true) AS `boolean`,
 
 === Description
 
+Type functions perform operations that check or convert expressions.
+
+=== Arguments
+
+expression:: [Required] The expression to check.
+
+=== Return Value
+
 Returns True if expression is a Boolean, number, or string, otherwise returns MISSING, NULL or false.
 
 === Examples
```

Comment:
The result of the example is still wrong. Look at the description of this function: we would expect that it will return `true` for `boolean`, `number`, and `string`; `null` for `null`; and `false` for `array` and `object`.